app/models/member.rb

```diff
@@ -5,7 +5,8 @@ class Member < ActiveRecord::Base
 
   scope :managers, conditions: { manager: true }
   scope :nonmanagers, conditions: 'manager IS NULL or manager = 0'
-
+  scope :recent_community_owner, ->(current_user) {where(user_id: current_user.id, manager: true,
```

Comment:
Line is too long. [97/80]<br>Use the `lambda` method for multi-line lambdas.<br>Block body expression is on the same line as the block start.